ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -226,60 +152,91 @@ The list displays all workflow versions of the selected workflow. Depending on t
+        Transition label defined by the user. It can be used to specify or simplify the transition name. When editing the transition label, it is changed both at the top of the settings sidebar and in the transition rectangle in the diagram.
 
     - *Queue type*  
-        Click the drop-down list to select the applicable queue type.   
+        Click the drop-down list to select the applicable queue type, see [Queue types](05_QueueTypes.md).
+
+    - *Priority*   
+        The *Priority* field is used in high performance scenarios. You can use it to speed up a critical path when there are so many actions to be processed that they cannot be processed in parallel, and they pile up in the backlog. You do this by prioritizing the actions that are in the critical path. For example, a critical path might be when an order has been received and needs to be sent to shipping, while other actions such as sending emails are not as time-critical. Note: Priority 10 increases the processing probability by a factor of 10 over priority 1, and priority 8 increases the processing probability by a factor of 4 over priority 2.
+
+    - *Max tries*   
+        Number of times the action execution should be retried after an error has occurred. This setting depends on the transition and is usually preconfigured. If you change this setting to a value greater than 1, you can select a retry time strategy from a drop-down list. Depending on this retry time strategy, the value in the *Strategy config value* field is used to perform the number of retries specified in the *Max tries* field. The following retry time strategies are available:
+
+        - **Constant**   
+            Define a constant value. For example, a constant value of 3 means to retry the action every 3 minutes.
+
+        - **CustomFunction**  
+            Use any suitable php function to define the retry time.
+
+        - **Custom**   
+            Enter the number of minutes, separated by commas, to define the time after which the action should be retried. For example, 2,7,9 means to retry after 2, 7, and 9 minutes, if you have entered **3** in the *Max tries* field.
+
+        - **Exponential**   
+            Enter an exponential function to define the amount of time the process should wait for the next execution of the retry. For example, 2^Y.
+
+        - **Linear**   
+            Enter a linear sequence for the process to wait for the next retry execution. For example, 1,2,3 means to retry after 1, 2, and 3 minutes, if you have entered **3** in the *Max tries* field.
+
+        - **Quadratic**   
+            Enter a square value to define the time in minutes after which the action should be retried, for example, Y^2.
+
+        Select the retry strategy and enter the required value in minutes in the *Strategy config value* field.
```

Comment:
Hier sollte eigentlich "Strategy config value" erklärt werden. Ich würde die drei Drop-down Lists separat auflisten un erklären. Ich denke, so wäre es einfacher zu verstehen.
- *Max tries*
(description) -> If greater than 1, Retry time strategy" drop-down list is displayed.
- *Retry time strategy*
(description with all options, wie oben) -> When retry time strategy selected, "Strategy config value" displayed.
- *Strategy config value*
(description)